Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -0,0 +1,79 @@
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+
+namespace Watchman.Discord.Areas.Responses.Services
+{
+    public class ResponsesMessageService
+    {
+        private const string DESCRIPTION = "dokumentacja:\nhttps://watchman.readthedocs.io/pl/latest/135-services-in-framework/";
+        private readonly ResponsesGetterService _responsesDatabase;
+        private readonly EmbedMessageSplittingService _embedMessageSplittingService;
+
+        public ResponsesMessageService(ResponsesGetterService responsesDatabase, EmbedMessageSplittingService embedMessageSplittingService)
+        {
+            _responsesDatabase = responsesDatabase;
+            _embedMessageSplittingService = embedMessageSplittingService;
+        }
+
+        public async Task PrintResponses(string commandArgument, Contexts contexts)
+        {
+            if (commandArgument == "default")
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Domyślne responses:", DESCRIPTION, GetDefaultResponses(), contexts);
+            }
+            else if (commandArgument == "custom")
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Nadpisane responses:", DESCRIPTION, GetCustomResponses(contexts.Server.Id), contexts);
+            }
+            else
+            {
+                await _embedMessageSplittingService.SendEmbedSplitMessage("Wszystkie responses:", DESCRIPTION, GetAllResponses(), contexts);
+            }
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetDefaultResponses()
+        {
+            return _responsesDatabase.GetResponsesFromBase()
+                .Where(x => x.IsDefault)
+                .Select(x => new KeyValuePair<string, string>(x.OnEvent, GetMessageWithoutFramesAndBold(x.Message)));
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetCustomResponses(ulong serverId)
+        {
+            var responses = _responsesDatabase.GetResponsesFromBase()
+                .Where(x => x.ServerId == serverId)
+                .Select(x => new KeyValuePair<string, string>(x.OnEvent, GetMessageWithoutFramesAndBold(x.Message)));
+
+            if (!responses.Any())
+            {
+                responses = responses.Append(new KeyValuePair<string, string>("-------------------------------", "```Obecnie jest brak nadpisanych responses!```"));
+            }
+            return responses;
+        }
+
+        private IEnumerable<KeyValuePair<string, string>> GetAllResponses()
+        {
+            return _responsesDatabase.GetResponsesFromBase()
+                .Select(x => new KeyValuePair<string, string>(x.OnEvent, GetMessageWithoutFramesAndBold(x.Message)));
+        }
+
+        private string GetMessageWithoutFramesAndBold(string message)
+        {
+            message = message.Contains('`')
+                ? message.Insert(message.IndexOf('`'), @"\")
```

Comment:
nie lepiej zrobić tu po prostu kilka replace?

Reply:
Replace zamienia wszystkie wystąpienia, a mi zależy żeby było tylko przed pierwszym. Wcześniej próbowałem przed wszystkimi, ale później te ukośniki były wszędzie widziane.

Reply:
a co jeśli będzie wiele tekstów, które będą w środku tych znaków?

Reply:
Teraz jak sprawdzałem, to z Replace też działa XD